packages/vue-components/src/Tab.vue

```diff
@@ -94,4 +94,18 @@ export default {
     .fade-leave-active {
         transition: opacity 0s;
     }
+
+    @media print {
+        .printable-tab-header {
+            display: block !important;
```

Comment:
I'm thinking maybe we can use the `d-print-block` class, since we are using bootstrap. And it may look a little better than having to use `!important` for css. However, I don't think it's going to impact much if we choose either, because we still need the print media query and classes here to set the styling for the pdf to look presentable. Just a personal suggestion :-)